libraries/aws_vpc_subnet.rb

```diff
@@ -6,42 +6,48 @@ class AwsVpcSubnet < Inspec.resource(1)
   name 'aws_vpc_subnet'
   desc 'This resource is used to test the attributes of a VPC subnet'
   example "
-    describe aws_vpc_subnet(vpc_id: 'vpc-12345678', subnet_id: 'subnet-12345678') do
+    describe aws_vpc_subnet(subnet_id: 'subnet-12345678') do
       it { should exist }
       its('cidr_block') { should eq '10.0.1.0/24' }
     end
   "
 
   include AwsResourceMixin
   attr_reader :vpc_id, :subnet_id, :cidr_block, :availability_zone, :available_ip_address_count,
-              :default_for_az, :map_public_ip_on_launch, :state, :ipv_6_cidr_block_association_set,
-              :assign_ipv_6_address_on_creation
+              :default_for_az, :mapping_public_ip_on_launch, :available, :ipv_6_cidr_block_association_set,
+              :assigning_ipv_6_address_on_creation
+  alias available? available
+  alias default_for_az? default_for_az
+  alias mapping_public_ip_on_launch? mapping_public_ip_on_launch
+  alias assigning_ipv_6_address_on_creation? assigning_ipv_6_address_on_creation
 
   def to_s
     "VPC Subnet #{@subnet_id}"
   end
+  
+  # Expose map_public_ip_on_launch and assign_ipv_6_address_on_creation
+  def method_missing(name)
```

Comment:
Method missing is a very powerful and dangerous tool - it's easy to hide bugs this way.  I'm also not sure about the motivation - the instance vars mentioned in the comment don't exist as instance variables.

Reply:
You are right. I had this as a solution before I found a better way and forgot to take it out.  But it is removed now!